common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -304,12 +304,9 @@ def __init__(self, *args, **kwargs):
             self._field_data.set_many(self, field_data)
             del self.data
 
-        editable_fields = super(VideoDescriptor, self).editable_metadata_fields
```

Comment:
Note to reviewers: These changes alone reduced local time of 22 secs to 10 secs for the VideoSummaryOutline endpoint on the 6.002 course.